src/autodiff_model.jl

```diff
@@ -100,11 +99,6 @@ function cons!(nlp :: ADNLPModel, x :: AbstractVector, c :: AbstractVector)
   return c
 end
 
-function jac(nlp :: ADNLPModel, x :: AbstractVector)
-  increment!(nlp, :neval_jac)
-  return ForwardDiff.jacobian(nlp.c, x)
-end
-
```

Comment:
Do we want the Jacobian to be sparse or dense here?

Reply:
We don't have sparse matrices with ForwardDiff, unfortunately.

Reply:
What I mean is that if you delete `jac()` here, then, the default `jac()` will be used and it creates a sparse matrix.

Reply:
Ah sorry, I see what you mean. I think we should return this then, so it's exactly what ForwardDiff gives.

Reply:
Fixed.